dds_web/__init__.py

```diff
@@ -102,11 +98,55 @@ def setup_logging(app):
                     "level": logging.DEBUG,
                     "propagate": False,
                 },
-                "actions": {"handlers": ["actions"], "level": logging.INFO, "propagate": False},
+                "actions": {
+                    "handlers": ["actions", "console"],
```

Comment:
Currently, action logging is also output to console for debugging purposes. Will remove that before merging!